zuul-core/src/main/java/com/netflix/zuul/util/HttpUtils.java

```diff
@@ -84,14 +85,23 @@ public static String extractClientIpFromXForwardedFor(String xForwardedFor) {
      * @return true if the content-encoding param containg gzip
      */
     public static boolean isGzipped(String contentEncoding) {
-        return contentEncoding.contains("gzip");
+        return contentEncoding.contains(HttpHeaderValues.GZIP.toString());
+    }
+
+    public static boolean isDeflated(String contentEncoding) {
```

Comment:
nit: Unrelated to your change, these should probably be `private`. The public interface should accept the actual headers and not the raw String(like in `isGzipped`)